scripts/load_env_variables.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
Can you update docs/developer.md and mentioned that this can be used for when u want to run it inside golang  IDE locally